lib/membrane_h264_plugin/parser/au_splitter.ex

```diff
@@ -52,7 +52,7 @@ defmodule Membrane.H264.Parser.AUSplitter do
 
   @type access_unit_t() :: list(NALu.t())
 
-  # split_nalus/5 defines a finite state machine with two states: :first and :second.
+  # split/5 defines a finite state machine with two states: :first and :second.
   # The state :first describes the state before reaching the primary coded picture NALu of a given access unit.
   # The state :second describes the state after processing the primary coded picture NALu of a given access unit.
```

Comment:
I know this is not the main topic of the PR, but these line are worth including into the documentation - perhaps moving description of `:first` and `:second` to the typedoc of `t()`

Reply:
I am not 100% convinced - `AUSplitter.t()` is opaque and I believe that the knowledge of what does particular state represent is worth only for those who are reading the implementation of the `split/2` function - therefore, as far as I am concerned, I wouldn't include that information in the public documentation.
WDYT @mickel8 ?